File: SoftuniadaTrainingContest-Sandbox/Min3Numbers.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Scanner;

public class MinNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        List<Integer> numbers = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            numbers.add(Integer.parseInt(scanner.nextLine()));
        }

        Collections.sort(numbers);

        for (int i = 0; i < Math.min(n, 3); i++) {
            System.out.println(numbers.get(i));
        }
    }
}

File: SoftuniadaTrainingContest-Sandbox/01.Min3Numbers/01.Min3Numbers/Min3Numbers.cs
using System;
using System.Collections.Generic;

namespace _01.Min3Numbers
{
    public class Min3Numbers
    {
        public static void Main(string[] args)
        {
            var n = int.Parse(Console.ReadLine());
            var numbers = new List<int>();

            for (int i = 0; i < n; i++)
            {
                numbers.Add(int.Parse(Console.ReadLine()));
            }

            numbers.Sort();

            for (int i = 0; i < Math.Min(n, 3); i++)
            {
                Console.WriteLine(numbers[i]);
            }
        }
    }
}
